apps/communications/contacts/js/contacts.js

```diff
@@ -304,6 +310,9 @@ var Contacts = (function() {
       if (activityName === 'pick' || activityName === 'update') {
         alternativeTitle = 'selectContact';
       }
+      var groupsList = document.getElementById('groups-list');
+      var dataType = ActivityHandler.activityDataType;
```

Comment:
better to rename this variable to activityType to be full consistent in the naming